recipes/turtleFSI/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "turtleFSI" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KVSLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 84adf1e4e7a927ee43bd8a18cab75dd08d4d5f21f4b98cac7b920bbe6525bfb9
+
+build:
+  number: 0
+  skip:
+    - True  # [win or py2k]
+  script: "{{ PYTHON }} setup.py install"
```

Comment:
Why not use pip?

Reply:
No particular reason, this is now changed to pip.